tools/deployment/k8s/s3/deploy.sh

```
@@ -0,0 +1,92 @@
+    fi
+}
+
+if [ -z "$DECOMPRESSED_ASSETS_DIRECTORY" ]; then
+    # If not defined, provide a default value
+    DECOMPRESSED_ASSETS_DIRECTORY="/tmp/yscope-log-viewer"
+fi
+
+# Download and extract YScope log viewer RELEASE tarball
+echo "------------------------------------------------------------------------------------------"
+echo "Downloading YScope log-viewer release from"
+echo ${RELEASE_URL}
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)